Content.Tests/DMTests.cs

```diff
@@ -26,9 +26,10 @@ public enum DMTestFlags {
         NoError = 0,        // Should run without errors
         Ignore = 1,         // Ignore entirely
         CompileError = 2,   // Should fail to compile
-        RuntimeError = 4,   // Should throw an exception at runtime
-        ReturnTrue = 8,     // Should return TRUE
-        NoReturn = 16,      // Shouldn't return (aka stopped by a stack-overflow or runtimes)
+        CompileWarning = 4, // Should throw a warning during compile
```

Comment:
We usually test for warnings by putting `#pragma EmissionCode error` inside the DM file. I would rather unimplemented warnings be given a code so we can do that instead.

Reply:
that makes sense, addressed